main.css

```diff
@@ -0,0 +1,223 @@
+
+/** Breakpoints:
+ * md > 768px
+ * lg > 992px
+ */
+
+/**
+ * Main theme
+ * ======================
+ **/
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 14px;
+  margin: 0;
+  padding: 0;
+}
+
+@media (min-width: 768px) {
+  body {
+    font-size: 16px;
+  }
+}
+
+@media (min-width: 992px) {
+  body {
+    font-size: 18px;
+  }
+}
+
+.t-bg-red {
+  background-color: #f72211;
+}
+
+.t-bg-gray {
+  background-color: #f4f5f6;
+}
+
+.t-bg-white {
+  background-color: white;
+}
+
+.t-color-red {
+  color: #f72211;
+  fill: #f72211;
+}
+
+.t-color-white {
+  color: white;
+}
+
+.t-color-black {
+  color: black;
+}
+
+.t-color-gray {
+  color: #807f84;
+}
+
+.t-color-darkgray {
+  color: #666;
+}
+
+.t-font-roboto {
+  font-family: 'Roboto Condensed', sans-serif;
+}
+
+.t-btn {
+  color: black;
+  font-weight: 700;
+  font-size: 13px;
+  background: #fee400;
+  border: none;
```

Comment:
- [ ] This can be done with the class `border-0`

Reply:
Thank you for your feedback!